.github/workflows/main.yml

```diff
@@ -79,6 +79,7 @@ jobs:
         set -e
         bash scripts/test/prepare.sh
         source build/env/bin/activate
+        docker pull sqlflow/sqlflow:step
```

Comment:
Do we need to use the step Docker image built from a PR?

Reply:
Yes, will do that later, all the test cases should run in the step image I think.